### ISSUE TITLE
Categorical distribution

### ISSUE THREAD

**Issue by xidulu** [justindomke/pangolin#9]
I suppose currently the categorical distribution only supports probability vector but not logit.

This is a little bit inconvenient in that softmax is also not implemented yet, as such it could be hard to implement multi-class classification model.

**Comment by justindomke** [comment]
softmax installed (for numpyro and stan, not obvious how to support it for JAGS)
